src/epoll.rs

```diff
@@ -85,7 +87,11 @@ impl Poller {
     }
 
     /// Adds a new file descriptor.
-    pub fn add(&self, fd: RawFd, ev: Event, mode: PollMode) -> io::Result<()> {
+    ///
+    /// # Safety
+    ///
+    /// The `fd` must be a valid file descriptor.
```

Comment:
It'd be good to also mention that the file descriptor must remain valid as long as it's in the `Poller`.

Reply:
This isn't true for `epoll` only. I've added documentation clarifying this.